grants_tagger/__main__.py

```diff
@@ -272,6 +281,7 @@ def wellcome_science(
         print(f"{output_path} exists. Remove if you want to rerun.")
     else:
         preprocess(input_path, output_path, text_cols, meta_cols)
+        create_label_binarizer(output_path, label_binarizer_output_path)
```

Comment:
if we are to add this here, this should be in its own if then else that checks if the label_binarizer_path exists

Reply:
my initial though was that this could be added in the dvc command but I can see now that a separate cli command would need to be added to give the user the ability to split so maybe better like that.